vendor/bat-native-ads/src/bat/ads/internal/experiments/experiments_util.cc

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/experiments/experiments_util.h"
+
+#include "base/metrics/field_trial.h"
+
+namespace ads {
+namespace experiments {
+
+bool IsActive(
+    const std::string& name) {
+  if (!base::FieldTrialList::Find(name)) {
```

Comment:
why are we wrapping calls for field trials and converting to Experiments? This seems like it unnecessarily confuses and complicates things and adds a lot of extra code that doesn't follow standard conventions

Reply:
resolved with refactor